src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -48,7 +48,7 @@ export const RadioGroup = <T, >({
     return options.map((option, index) => (
       <Radio
         description={option.description}
-        idDisabled={option.disabled}
+        isDisabled={option.disabled}
         // eslint-disable-next-line react/no-array-index-key
         key={index}
```

Comment:
I would allow an `option.key` and use it if existing, fallbacking to the array index if not

Reply:
good idea. done